cli/azd/pkg/infra/provisioning/provider.go

```diff
@@ -19,9 +19,10 @@ const (
 )
 
 type Options struct {
-	Provider ProviderKind `yaml:"provider,omitempty"`
-	Path     string       `yaml:"path,omitempty"`
-	Module   string       `yaml:"module,omitempty"`
+	Provider         ProviderKind   `yaml:"provider,omitempty"`
+	Path             string         `yaml:"path,omitempty"`
+	Module           string         `yaml:"module,omitempty"`
+	DeploymentStacks map[string]any `yaml:"deploymentStacks,omitempty"`
```

Comment:
Would `terraform` know about `deploymentStacks` here in provisioning_provider.go? I think we would want a generic-props that is translated into a structured-props. See example of `RawProps` vs `Props` an implementation [here](https://github.com/Azure/azure-dev-pr/blob/b5f96606635e53819ee4fdb1a21337c9642bd622/cli/azd/pkg/project/resources.go#L84).

Reply:
No, terraform would not know or care about deployment stacks since it does not leverage ARM based deployments.